db/schema.rb

```diff
@@ -503,4 +455,52 @@
     GROUP BY rejected.prison_name, rejected.reason, round((((rejected.rejected_count)::numeric / (booked.booked_count)::numeric) * (100)::numeric), 2), rejected.year, rejected.week;
   SQL
 
+  create_view :timely_and_overdues,  sql_definition: <<-SQL
+      SELECT count(*) AS count,
+      'overdue'::text AS status,
+      vsc.visit_state,
+      prisons.name AS prison_name
+     FROM ((visits v
+       JOIN prisons ON ((prisons.id = v.prison_id)))
+       JOIN visit_state_changes vsc ON (((v.id = vsc.visit_id) AND ((vsc.visit_state)::text = ANY ((ARRAY['booked'::character varying, 'rejected'::character varying, 'cancelled'::character varying, 'withdrawn'::character varying])::text[])))))
```

Comment:
I wonder if blacklisting might be better here in case we add a new state. I think we want to exclude the 'requested' state so might be safer to use that in the view?